front-end/src/components/NetworkInfo.tsx

```diff
@@ -28,45 +28,37 @@ const NetworkInfo = ({ className }: Props) => {
 			<div className="text">Join our Community to discuss, contribute and get regular updates from us!</div>
 			{!error && data &&
 				<div className="networkInfo-icons">
-					{ data.blockchain_socials[0].homepage &&
-						<a href={data.blockchain_socials[0].homepage} target='_blank' rel='noreferrer'>
-							<Icon name='home'/>
-						</a>
+					{data.blockchain_socials[0].homepage && <a href={data.blockchain_socials[0].homepage} target='_blank' rel='noreferrer'>
```

Comment:
This will throw error if data.blockchain_socials[0] is null

Reply:
Adding additional check.